docs/topics/api/auth_internal.rst

```diff
@@ -39,6 +39,35 @@ The token is available in two forms:
     * For all endpoints, as a cookie called ``api_auth_token``. This cookie
       expires after 30 days and is set as ``HttpOnly``.
 
+The response will contain some profile data for personalization:
+
+    :>json int id: The numeric user id.
+    :>json string email: Email address used by the user to log in and create this account.
+    :>json string name: The name chosen by the user, or the username if not set.
+    :>json string picture_url: URL to a photo of the user, or `/static/img/anon_user.png` if not set.
+    :>json string username: username chosen by the user, used in the account url. If not set will be a randomly generated string.
```

Comment:
If the user hasn't set a username yet and this is random, is there any way of knowing that? It would be great to show "New User" or something rather than "user1694942" on the page. That's likely unrelated to the this patch though...